### ISSUE TITLE
automatic_make.sh cd failing due to missing ..

### ISSUE THREAD

**Issue by Christian-B** [SpiNNakerManchester/SpiNNakerManchester.github.io#15]
The file found at:
http://spinnakermanchester.github.io/development/automatic_make.sh
Is buggy
some of the cd commands are failing due to missing ..  (up one level)

Working script (almost) provided by @alan-stokes .

cd spinnaker_tools
source setup
make clean
make || exit $?
cd ..
cd spinn_common
make clean
make || exit $?
make install
cd ..
cd SpiNNMan/c_models/reinjector/
make clean
make || exit $?
cd ../../..
cd SpiNNFrontEndCommon/c_common/
cd front_end_common_lib/
make install-clean
cd ..
make clean
make || exit $?
make install
cd ../..
cd sPyNNaker/neural_modelling/
make clean
make || exit $?
source setup
cd ../../sPyNNakerExternalDevicesPlugin/neural_modelling/
make clean
make || exit $?
cd ../../SpiNNakerGraphFrontEnd/spinnaker_graph_front_end/examples/
make clean
make || exit $?
cd ../../../sPyNNakerExtraModelsPlugin/neural_modelling
make clean
make || exit $?
echo "completed"

**Comment by rowleya** [comment]
Oops, sorry about that - I think I merged together two scripts to get this one!

One though would be that this could itself be made into a Makefile.  If we made sure that the neural code has the appropriate header files as dependencies thus causing a re-make when certain headers change, it would probably possible to never actually need to call "make clean" in general, and so a top-level (though optional) Makefile would work well.  This would also make it easier when switching branches - calling make as part of the procedure would then avoid issues with out-of-date binaries (which does occur on occasion).
